stable/sumologic-fluentd/templates/daemonset.yaml

```diff
@@ -229,16 +229,16 @@ spec:
         {{- end }}
     {{- if .Values.tolerations }}
       tolerations:
-{{ toYaml .Values.tolerations | indent 8 }}
+    {{ toYaml .Values.tolerations | indent 8 }}
     {{- end }}
     {{- if .Values.nodeSelector }}
       nodeSelector:
-{{ toYaml .Values.nodeSelector | indent 8 }}
+    {{ toYaml .Values.nodeSelector | indent 8 }}
     {{- end }}
     {{ if .Values.affinity }}
       affinity:
-{{ toYaml .Values.affinity | indent 8 }}
-      {{- end }}
+    {{ toYaml .Values.affinity | indent 8 }}
```

Comment:
This alignment, is causing helm to throw this error at run time.

nd error message: Error: YAML parse error on sumologic-fluentd/templates/daemonset.yaml: error converting YAML to JSON: yaml: line 96: did not find expected key
[0m

Manually editing chart version 1.0.0 with nodeSelector works just fine. Can you please change the alignment to what it was before this commit - b48afcb3c4e7f9f6e450a3884b448dca5b41ddd0